docs/src/examples/banana.md

```diff
@@ -0,0 +1,108 @@
+# Banana shaped distribution
```

Comment:
Thanks @burtonjosh for this excellent contribution! It looks like the text has a lot of overlap with the Quickstart. Currently the Quickstart jumps from a "trivial" example where Pathfinder succeeds (MvNormal) to a really hard example where Pathfinder fails (Neal's funnel). I think this would be an excellent transition. Would you mind adapting the Quickstart to make this the second example instead of its own page?

Reply:
Just done this, I agree it's a nice transition. Let me know what you think!